### PR TITLE
added basic business card template

### PR DESCRIPTION
**Pull request by nikhiltripathi1**
I have added a basic business card widget
Desktop view

image

image

mobile view

image

image

### CODE REVIEW

**Comment by nikhiltripathi1** [comment]
@WebDevRaj 
@ManojSatishkumar 
@venkateshpachigulla 
@RitikPandey1 
@AKAspanion

File: src/modules/business-card/basic/variants/back side/render.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+  return (
+    <>
+    <div style={{
+      ...styles.root,
+      backgroundColor: "black",
+      color: '#e8d743'
+    }}>
+      <div style={{...styles.mainContent}}>
```
Comment (RitikPandey1) [inline src/modules/business-card/basic/variants/back side/render.js:33]: no need of spread operator

File: src/modules/business-card/basic/variants/back side/render.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  }
+) => {
+  const styles = useStyles()
+
+
+  return (
+    <>
+    <div style={{
+      ...styles.root,
+      backgroundColor: "black",
```
Comment (RitikPandey1) [inline src/modules/business-card/basic/variants/back side/render.js:30]: use theme colours only

File: src/modules/business-card/basic/variants/front side/render.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <>
+    <div style={{
+      ...styles.root,
+      backgroundColor: 'black',
+      color: theme.value.contrast
+    }}>
+      <div style={{
+        ...styles.mainContent,
+      backgroundColor: '#e8d743',
```
Comment (RitikPandey1) [inline src/modules/business-card/basic/variants/front side/render.js:34]: use theme colours only

File: src/modules/business-card/basic/variants/back side/render.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      color: '#e8d743'
```
Comment (sammjainn) [inline src/modules/business-card/basic/variants/back side/render.js:31]: use theme colors instead of hexadecimal value

**Comment by nikhiltripathi1** [comment]
resolved all the above mentioned issues @RitikPandey1 @sammjainn

File: src/modules/business-card/basic/variants/index.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import Backside from './back side/index.js'
+import Frontside from './front side/index.js'
+
+export default {
+  Backside,
+  Frontside
+}
```
Comment (AKAspanion) [inline src/modules/business-card/basic/variants/index.js:7]: Format all files with prettier.

Comment (nikhiltripathi1) [reply src/modules/business-card/basic/variants/index.js]: Done, please review it now.

**Pull request merged**